cmake/compiler_settings.cmake

```diff
@@ -16,6 +16,8 @@ if (NOT CMAKE_C_COMPILER_ID STREQUAL CMAKE_CXX_COMPILER_ID)
     "${CMAKE_C_COMPILER_ID} != ${CMAKE_CXX_COMPILER_ID}")
 endif ()
 
+set(CMAKE_CXX_STANDARD 99)
```

Comment:
Doesn't this change to 14 three lines further down?

Reply:
whoops, this should be CMAKE_C_STANDARD